data/mods/Russ_Extra_Stuff/construction.json

```diff
@@ -0,0 +1,30 @@
+[{
+        "type" : "construction",
```

Comment:
I think you could have it in the same option as regular "Start vehicle". Just overwrite it and add that one component.